### PR TITLE
ObjectJsonAdapter should not fail on null values.

### LINKED ISSUE #79: Parsing fails on NULL field values
This JSON
{
		"id":"2",
		"name":"glpi",
		"firstname":null
}

fails: Expected a value but was NULL at path $.firstname

class ObjectJsonAdapter  in  file StandardJsonAdapters.java
has switch (reader.peek()) {}
there are BEGIN_ARRAY, BEGIN_OBJECT, STRING, NUMBER, BOOLEAN
and noooooo  NULL value  among  cases !!!

### PR DESCRIPTION
**Pull request by serj-lotutovici**
Fixes #79

### CODE REVIEW

File: moshi/src/test/java/com/squareup/moshi/ObjectAdapterTest.java [hunk 1]
```diff
@@ -84,6 +85,17 @@
     assertThat(adapter.fromJson("[0, 1]")).isEqualTo(Arrays.asList(0d, 1d));
   }
 
+  @Test public void fromJsonDoesNotFailOnNullValues() throws Exception {
+    Moshi moshi = new Moshi.Builder().build();
+    JsonAdapter<Object> adapter = moshi.adapter(Object.class);
+
+    try {
+      adapter.fromJson("{\"address\":null, \"items\":null}");
```
Comment (swankjesse) [inline moshi/src/test/java/com/squareup/moshi/ObjectAdapterTest.java]: nit: the try/catch isn't really necessary. Better would be to confirm that the result matches a map with two keys whose values are null

**Comment by swankjesse** [comment]
Nice! Three things:

1. Please complete our contributor's agreement. https://github.com/square/moshi/blob/master/CONTRIBUTING.md
2. Please adjust the test to confirm that the null values are retained.
3. Please use http://rebaseandsqua.sh when you're done that so we get a single commit in our history. Then push the change again.

Thanks!

**Comment by serj-lotutovici** [comment]
> Please adjust the test to confirm that the null values are retained.

Done

> Please complete our contributor's agreement. https://github.com/square/moshi/blob/master/CONTRIBUTING.md

Signed the CLA a few minutes before submitting the pr.

**Comment by swankjesse** [comment]
Awesome!

**Pull request merged**